### PR TITLE
Add templateContent() directive

### PR DESCRIPTION
**Pull request by justinfagnani**
This is a generically useful utility. It has been useful at Google for rendering SVG icons rendered to generated templates, for instance.

### CODE REVIEW

File: src/directives/template-content.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * Renders the content of a template as HTML.
+ *
+ * Note, this trusts the template contents to be developer controlled and not
+ * user controlled. User controlled templates rendered with this directive
+ * could lead to XSS vulnerabilities.
+ */
+export const templateContent =
+    directive((template: HTMLTemplateElement) => (part: Part): void => {
+      if (!(part instanceof NodePart)) {
+        throw new Error('unsafeHTML can only be used in text bindings');
```
Comment (web-padawan) [inline src/directives/template-content.ts]: Copy-paste leftover: `unsafeHTML`

File: src/directives/template-content.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
```
Comment (web-padawan) [inline src/directives/template-content.ts]: Maybe 2020?

File: src/test/directives/template-content_test.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
```
Comment (web-padawan) [inline src/test/directives/template-content_test.ts]: This could be 2020 too.

File: src/directives/template-content.ts (hunk 2 above)
Comment (justinfagnani) [reply src/directives/template-content.ts]: 👍

File: src/test/directives/template-content_test.ts (hunk 3 above)
Comment (justinfagnani) [reply src/test/directives/template-content_test.ts]: 👍

**Comment by justinfagnani** [comment]
@sorvell  I considered `stampTemplate`, but "stamp" seems a little informal and Polymer-team lingo. `cloneTemplate` could be it too. Internally the directive is named just `content`, but I thought that wasn't specific enough.

File: src/directives/template-content.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import {directive, NodePart, Part} from '../lit-html.js';
+
+interface PreviousValue {
+  readonly template: HTMLTemplateElement;
+  readonly fragment: DocumentFragment;
+}
+
+// For each part, remember the value that was last rendered to the part by the
+// templateContent directive, and the DocumentFragment that was last set as a
+// value. The DocumentFragment is used as a unique key to check if the last
```
[the remark this answers is not in the document]
Comment (justinfagnani) [reply src/directives/template-content.ts]: See the 're-renders a template over a non-templateContent value' test

**Comment by justinfagnani** [comment]
@jridgewell just did

**Pull request merged**